src/plugins/plugin.tooltip.js

```diff
@@ -353,13 +355,13 @@ export class Tooltip extends Element {
 
     this.opacity = 0;
     this._active = [];
-    this._chart = config._chart;
     this._eventPosition = undefined;
     this._size = undefined;
     this._cachedAnimations = undefined;
     this._tooltipItems = [];
     this.$animations = undefined;
     this.$context = undefined;
+    this.chart = config.chart;
```

Comment:
I think we need to do it this way, in case someone is using the Tooltip constructor directly in their project:
```js
// TODO: V4, remove this backward compatibility fallback to _chart
this.chart = config.chart || confgi._chart;
`